src/components/sidebar/sidebar.css

```diff
@@ -0,0 +1,32 @@
+
+.sidebar-section {
+    width: 24rem;
+    padding-top: 5rem;
+    background-color: var(--v2-bg-color);
+    height: 100vh;
+    position: fixed;
+}
+
+.sidebar-item {
+    color: var(--text-color);
+    display: flex;
+    align-items: center;
+    font-size: 2.2rem;
+    padding: 0.8rem;
+    padding-left: 2.8rem;
+    text-decoration: none;
+}
+
+.active-sidebar-item:hover {
+    background-color: var(--v1-bg-color);
+}
+
+.sidebar-item:focus {
+    background-color: var(--v1-bg-color);
```

Comment:
You can merge these two `.sidebar-item:focus, sidebar-item:focus`

Reply:
Thanks, @ApurvaSawant11  for pointing this out.